### PR TITLE
[000] Add contributor in script js file

### PR DESCRIPTION
**Pull request by nazma98**
### QA Steps

- [ ]

### CODE REVIEW

**Review by Tamjidul-Islam — APPROVED** [review]

File: script.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ const contributors = [
     dirPath: 'bongodev',
     email: 'support@bongodev.com',
   },
+  {
+    name: 'Nazma',
+    dirPath: 'nazma98',
+    email: 'nazma.sarker1398@gmail.com'
+  }
```
Comment (Armstrong104) [inline script.js:34]: Can we please add a comma ',' after 2nd curly braces in line 34 so that we don't get conflict.

**Review by Armstrong104 — CHANGES_REQUESTED** [review]
Please change according to the review. Thank you.

File: script.js (hunk 1 above)
Comment (nazma98) [reply script.js]: Thanks for pointing it out! I've fixed it.

Comment (Armstrong104) [reply script.js]: Welcome. Keep doing the good work.

**Review by Armstrong104 — APPROVED** [review]

**Pull request merged**